lambdas/search/index.py

```diff
@@ -20,6 +20,40 @@ def lambda_handler(request):
     """
     Proxy the request to the elastic search.
     """
+    action = request.args.get('action')
+    index = request.args.get('index')
+
+    if action == 'search':
+        query = request.args.get('query', '')
+        body = {
+            "query": {
+                "simple_query_string" : {
+                    "query": query,
+                    "fields": ['content', 'comment', 'key_text', 'meta_text'],
+                    "default_operator": "or"
+                }
+            }
+        }
+        # TODO: should be user settable; we should proably forbid `content` (can be huge)
```

Comment:
Are these the fields that Elastic will search to find matches? Or, the fields that are returned in the result? Either way, don't we want the document contents?

Reply:
Fields that get returned, I think... This was a comment from @akarve's example

Reply:
Yes, this prevents Elastic from sending back unnecessary huge stuff, e.g. `content`